### PR TITLE
show fault domain in node and task subcommand output

### PR DESCRIPTION
**Pull request by darkonie**
Add 2 new fields: `REGION` and `ZONE` to `dcos node` and `dcos task` output

[DCOS-17719](https://jira.mesosphere.com/browse/DCOS-17719)

screen shot 2017-09-27 at 3 25 34 pm

### CODE REVIEW

File: dcos/util.py [hunk 1]
```diff
@@ -726,3 +726,19 @@ def read_file_json(path):
...
             return load_json(options_file)
+
+
+def get_fault_domain(state):
+    """ Read mesos state json and return region and zone
+
+    :param state: mesos state json top level or a node item
+    :type state: dict
+    :returns: region and zone
+    :rtype: str | None
```
Comment (bamarni) [inline dcos/util.py]: As this returns a tuple and not a single value, should this be `:rtype: (str | None, str | None)`?

Comment (darkonie) [reply dcos/util.py]: fixed

**Comment by darkonie** [comment]
run all integration tests

**Comment by darkonie** [comment]
run all integration tests

**Comment by darkonie** [comment]
run all integration tests

**Pull request merged**